tests/unit/scanners/test.javascript.js

```diff
@@ -263,9 +218,8 @@ describe('JavaScript Scanner', () => {
     expect(ok).toBeTruthy();
   });
 
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
This doesn't seem needed anymore, this test isn't disabled anymore in this version of the patch.

Reply:
:/